### PR TITLE
Feature/128190 meta data dal in coordinator

### PR DESCRIPTION
**Pull request by ftoft**
## Description

Implementation of DAL for MetaData

## References

* #0000

### CODE REVIEW

File: source/coordinator/GreenEnergyHub.Aggregation.DatabaseMigration/001_JobsTable.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System;
 using FluentMigrator;
```
Comment (kristianschneider) [inline source/coordinator/GreenEnergyHub.Aggregation.DatabaseMigration/001_JobsTable.cs:16]: We might need to wait for the DbUp PR to get into main which is the new way to create the schemas

File: source/coordinator/GreenEnergyHub.Aggregation.Infrastructure/MetaDataDataAccess.cs [hunk 2]
```diff
@@ -0,0 +1,158 @@
+﻿// Copyright 2020 Energinet DataHub A/S
```
Comment (jonasdmoeller) [inline source/coordinator/GreenEnergyHub.Aggregation.Infrastructure/MetaDataDataAccess.cs:1]: Using parameters 'job' and 'result' can lead to null pointer exceptions

Comment (jonasdmoeller) [inline source/coordinator/GreenEnergyHub.Aggregation.Infrastructure/MetaDataDataAccess.cs:1]: await all async calls

Comment (jonasdmoeller) [inline source/coordinator/GreenEnergyHub.Aggregation.Infrastructure/MetaDataDataAccess.cs:1]: Configure await when awaiting async calls

Comment (jonasdmoeller) [inline source/coordinator/GreenEnergyHub.Aggregation.Infrastructure/MetaDataDataAccess.cs:1]: await using statements

File: source/coordinator/GreenEnergyHub.Aggregation.Tests/InputProcessorTests.cs [hunk 3]
```diff
@@ -37,7 +39,7 @@ public InputProcessorTests()
             _strategy = Substitute.For<IDispatchStrategy>();
             _strategy.FriendlyNameInstance.Returns(StrategyName);
             list.Add(_strategy);
-            _inputProcessor = new InputProcessor(Substitute.For<ILogger<InputProcessor>>(), list);
+            _inputProcessor = new InputProcessor(Substitute.For<ILogger<InputProcessor>>(), list, Substitute.For<IMetaDataDataAccess>());
         }
 
         [Fact]
```
Comment (jonasdmoeller) [inline source/coordinator/GreenEnergyHub.Aggregation.Tests/InputProcessorTests.cs:45]: We should rename unit test cases to follow the given-when-then pattern, i.e. given an unknown stragegy, when process input is called, an exception is thrown. So method name would be something like:
`public async Task UnknownStrategy_ProcessInput_ExceptionIsThrow()` or something like that..

**Review by jonasdmoeller — CHANGES_REQUESTED** [review]

**Comment by jonasdmoeller** [comment]
@kristianschneider Why was this pull request merged before review approval?

**Pull request merged**